src/components/pages/LandingPage.vue

```diff
@@ -182,35 +182,35 @@ export default {
       carouselData: 0,
       step_number: 0,
       workflow_steps: [
-        { text: 'Check case' },
+        { text: 'Review case' },
         { text: 'Select phenotypes' },
-        { text: 'Investigate variants' },
-        { text: 'Review findings' },
+        { text: 'Review variants' },
+        { text: 'Report findings' },
```

Comment:
Hi @tonydisera , 
Can you test once again to see if the wordings are synced? I cloned the repo to verify and seems to be showing the updated text. Also, you can see here that the text is changed. 

<img width="1538" alt="Screen Shot 2020-03-31 at 10 34 57 AM" src="https://user-images.githubusercontent.com/16284713/78057268-4e2c1d00-733b-11ea-9512-b7750ddd7b5c.png">